code/modules/library/lib_items.dm

```diff
@@ -2,6 +2,8 @@
 #define BOOKCASE_ANCHORED 1
 #define BOOKCASE_FINISHED 2
 
+GLOBAL_LIST_EMPTY(roundstart_books_by_area)
```

Comment:
```suggestion
///contains information on book info datums by area type.
///associative list of the form: list(area typepath = list([/datum/book_info] instances inside that area))
GLOBAL_LIST_EMPTY(roundstart_books_by_area)
```
or something to that effect, we have enough undocumented global lists

Reply:
Moved to the library subsystem, and documented a bit better